calico/maintenance/troubleshoot/commands.md

```diff
@@ -251,34 +251,20 @@ items:
 
 #### Run commands across multiple nodes
 
-Export THE_COMMAND_TO_RUN=date && for calinode in:
-
 ```bash
-kubectl get pod -o wide -n calico-system | grep calico-node | awk '{print $1}'`; do echo $calinode; echo "-----"; 
-```
+export THE_COMMAND_TO_RUN=date && for calinode in `kubectl get pod -o wide -n calico-system | grep calico-node | awk '{print $1}'`; do echo $calinode; echo "-----"; kubectl exec -n calico-system $calinode -- $THE_COMMAND_TO_RUN; printf "\n"; done
```

Comment:
This is already OK, but I think it would be even clearer if laid out in the conventional multi-line form, i.e.:
```
export THE_COMMAND_TO_RUN=date

for calinode in `kubectl get pod -o wide -n calico-system | grep calico-node | awk '{print $1}'`; do
    echo $calinode
    echo "-----"
    kubectl exec -n calico-system $calinode -- $THE_COMMAND_TO_RUN
    printf "\n"
done
```